qcodes/instrument_drivers/ZI/ZIUHFLI.py

```diff
@@ -1565,7 +1565,8 @@ def __init__(self, name: str, device_ID: str, **kwargs) -> None:
                            val_mapping = {'ON': 1, 'OFF': 0},
                            vals=vals.Enum('ON', 'OFF'))
 
-        # make this a slave parameter off scope_holdoff_seconds
+        # TODO: make scope_trig_holdoffmode change according to
+        # parameter off scope_holdoff_seconds
         # and scope_holdoff_events
         self.add_parameter('scope_trig_holdoffmode',
```

Comment:
I suggest removing todo since this is anyway deprecated so it will never be implemented